ref(node): Clarify usage of onFatalError option in node (#2643)
getsentry/sentry-docs: src/platforms/node/common/configuration/integrations/default-integrations.mdx

_Import name: `Sentry.Integrations.OnUncaughtException`_

This integration attaches a global uncaught exception handler. It can be modified to provide a custom shutdown function.

> The `onFatalError` option is meant to perform a cleanup before the process exits, not fully prevent it.

Reviewer: ```suggestion
The `onFatalError` option is meant to perform a cleanup before the process exits, not fully prevent it from exiting.
```

<Alert level="warning" title="Note">

> Be aware that if you are overwriting this setting, you will lose the default implementation, which handles draining queued events before exiting. If you want to see how it works and recreates in your custom code, see the implementation of [`logAndExitProcess`](https://github.com/getsentry/sentry-javascript/blob/master/packages/node/src/handlers.ts) function.

Reviewer: ```suggestion
Be aware that if you overwrite this setting, you will lose the default implementation, which handles draining queued events before exiting. If you want to see how it works in order to be able to recreate it in your code, see the implementation of the [`logAndExitProcess`](https://github.com/getsentry/sentry-javascript/blob/master/packages/node/src/handlers.ts) function.
```

> Be aware that if you are overwriting this setting, you will lose the default implementation, which handles draining queued events before exiting. If you want to see how it works and recreates in your custom code, see the implementation of [`logAndExitProcess`](https://github.com/getsentry/sentry-javascript/blob/master/packages/node/src/handlers.ts) function.

Reviewer 2: ```suggestion
Be aware that if you are overwriting this setting, you will lose the default implementation, which handles draining queued events before exiting. If you want to examine how it works and then recreate it in your code, see the implementation of the [`logAndExitProcess`](https://github.com/getsentry/sentry-javascript/blob/master/packages/node/src/handlers.ts) function.
```

> Be aware that if you overwrite this setting, you will lose the default implementation, which handles draining queued events before exiting. If you want to see how it works in order to be able to recreate it in your code, see the implementation of the [`logAndExitProcess`](https://github.com/getsentry/sentry-javascript/blob/master/packages/node/src/handlers.ts) function.

Author: ```suggestion
Be aware that if you overwrite this setting, you will lose the default implementation, which handles draining queued events before exiting. If you want to examine how it works in order to be able to recreate it in your code, see the implementation of the [`logAndExitProcess`](https://github.com/getsentry/sentry-javascript/blob/master/packages/node/src/handlers.ts) function.
```
